test/decaffeinate_test.js

```diff
@@ -391,6 +391,91 @@ describe('automatic conversions', function() {
       check(`(a) -> a = 1`, `(function(a) { return a = 1; });`);
     });
 
+    it('does add the default value when they is any', function() {
+      check(`(a=2) -> a`, `(function(a=2) { return a; });`);
+    });
+
+    it.skip('does work with if then throw inline', function() {
```

Comment:
All `it.skip` are features that I found were not present (switch case doesn't work, throw inline doesn't work, for i in range doesn't work, continue statement does not work). I didn't really know how to implement them, so I thought it would be already something to include them so that it is easier to fix those bugs.